libfs/unstage_file.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libfs
+
+import (
+	"bytes"
+
+	"github.com/keybase/client/go/logger"
+	"github.com/keybase/kbfs/libkbfs"
+	"golang.org/x/net/context"
+)
+
+// UnstageForTesting unstages all unmerged commits and fast-forwards
+// to the current master, if the given data is non-empty. If the given
+// data is empty, it does nothing. If the given data begins with the
+// bytes "async", the unstaging is done asynchronously, i.e. this
+// function returns immediately and the unstaging happens in the
+// background. (Other subsequent IO operations may be blocked,
+// though.)
```

Comment:
Maybe mention that: a) this should only be needed if there is a bug in automatic CR, and b) you can figure out when the unstage succeeds by polling `.kbfs_status`?